new-docs/examples/python/getting-started/application/expenses-flask/app/authorization.py

```diff
@@ -21,22 +21,14 @@ def authorize(action, resource):
     else:
         raise Forbidden("Not Authorized!")
 
-
```

Comment:
We'll need to make these same changes to https://github.com/osohq/oso-flask-tutorial, which is what we direct readers to clone / follow along with.

Actually, might make the most sense to replace `new-docs/examples/python/getting-started/application/expenses-flask` with https://github.com/osohq/oso-flask-tutorial as a submodule. What do you think?

Reply:
I agree! There's no sense in having to keep those manually in sync when we could use a submodule. I'll go ahead and write up a commit for that — would you like that on this PR or separately?

Reply:
This PR would be good.